src/v2/Apps/Authentication/authenticationRoutes.tsx

```diff
@@ -9,14 +21,80 @@ const ResetPasswordRoute = loadable(
   { resolveComponent: component => component.ResetPasswordRoute }
 )
 
+const LoginRoute = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "authenticationBundle" */ "./Routes/LoginRoute"
+    ),
+  { resolveComponent: component => component.LoginRoute }
+)
+
+const SignupRoute = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "authenticationBundle" */ "./Routes/SignupRoute"
+    ),
+  { resolveComponent: component => component.SignupRoute }
+)
+
+const runAuthMiddleware = compose(checkForRedirect, setReferer)
```

Comment:
is there a lodash function like `compose` that instead of threading args through functions that return an output, simply invokes said functions with initial args?

Reply:
You're using styled-system's compose here which... I don't think is going to work how you expect it to.

Are you looking for `curry`?

Reply:
Nah, compose is just an exposed helper function that composes two functions via curry. It works as one would expect. I don't know why lodash doesn't have it; it has flow, which is the same, only the other direction. I'm just looking for a way to say `invoke([fnA, fnB], withArgs)` -- something like that.

Reply:
It's not though — https://github.com/styled-system/styled-system/blob/6d822f27a486e5e2bc1cae1f44dd151ce874a863/packages/core/src/index.js#L186-L190 - it news up some parsers etc.

But yeah as you said there's flow and also flowRight